content/pt_PT/faq.md

```diff
@@ -1,34 +1,31 @@
 # FAQ
 
-## What is io.js?
+## O que é o io.js?
 
-[io.js](https://github.com/iojs/io.js) is a JavaScript platform built on [Chrome's V8 runtime](http://code.google.com/p/v8/). This project began as a fork of [Joyent's Node.js™](https://nodejs.org/) and is compatible with the [npm](https://www.npmjs.org/) ecosystem.
+O [io.js](https://github.com/iojs/io.js) é uma plataforma JavaScript construída sob o [runtime do Chrome V8](http://code.google.com/p/v8/). Este projeto começou como um fork do [Node.js™ da Joyent](https://nodejs.org/) e é compatível com o ecossistema do [npm](https://www.npmjs.org/).
 
-Why? io.js aims to provide faster and predictable release cycles. It currently merges in the latest language, API and performance improvements to V8 while also updating libuv and other base libraries.
+Porquê? O io.js procura disponibilizar ciclos de release mais rápidos e previsíveis. Atualmente junta as melhorias mais recentes da linguagem, API e performance do V8 e ao mesmo tempo atualiza a libuv e outras bibliotecas base.
 
-This project aims to continue development of io.js under an "[open governance model](https://github.com/iojs/io.js/blob/v1.x/GOVERNANCE.md#readme)" as opposed to corporate stewardship.
+Este projeto procura continuar o desenvolvimento do io.js sob um "[modelo de governança aberta](https://github.com/iojs/io.js/blob/v1.x/GOVERNANCE.md#readme)", ao invés de uma administração corporativa.
 
-## Version 1.0.x?
+## Versão 1.0.x?
 
-io.js has moved to [Semver](http://semver.org/) and the changes between Node.js™ 0.10 and io.js 1.0.0 were significant enough
-to warrant a major version increment.
+O io.js passou a usar [Semver](http://semver.org/) e as alterações entre o Node.js™ 0.10 e o io.js 1.0.0 foram significativas o suficiente para justificar um incremento na versão.
 
-Our [CHANGELOG](https://github.com/iojs/io.js/blob/v1.x/CHANGELOG.md) for v1.x provides a [summary of changes from Node.js v0.10.35 to io.js v1.0.x](https://github.com/iojs/io.js/blob/v1.x/CHANGELOG.md#summary-of-changes-from-nodejs-v01035-to-iojs-v100).
+O nosso [CHANGELOG](https://github.com/iojs/io.js/blob/v1.x/CHANGELOG.md) da v1.x disponibiliza um [resumo das alterações do Node.js v0.10.35 para o io.js v1.0.x](https://github.com/iojs/io.js/blob/v1.x/CHANGELOG.md#summary-of-changes-from-nodejs-v01035-to-iojs-v100).
 
-## How can I contribute?
+## Como posso contribuir?
 
-Everyone can help. io.js adheres to a [code of conduct](https://github.com/iojs/io.js/blob/v1.x/CONTRIBUTING.md#code-of-conduct), and contributions, releases, and contributorship are under an [open governance](https://github.com/iojs/io.js/blob/v1.x/GOVERNANCE.md#readme) model.
+Todos podem ajudar. O io.js adere a um [código de conduta](https://github.com/iojs/io.js/blob/v1.x/CONTRIBUTING.md#code-of-conduct), e as contribuições, lançamentos e autoria estão sob um modelo de [governança aberta](https://github.com/iojs/io.js/blob/v1.x/GOVERNANCE.md#readme).
```

Comment:
Not sure if I agree with the translation of "adheres" but I don't have a better pick. :)